cvxpy/lin_ops/canon_backend.py

```diff
@@ -1103,6 +1090,305 @@ def _to_dense(x):
         return res
 
 
+class StackedSlicesBackend(PythonCanonBackend):
+
+    @staticmethod
+    def reshape_constant_data(constant_data: Any, lin_op_shape: tuple[int, int]) \
+            -> dict[int, sp.csc_matrix]:
+        return {k: StackedSlicesBackend._reshape_single_constant_tensor(v, lin_op_shape)
+                for k, v in constant_data.items()}
+
+    @staticmethod
+    def _reshape_single_constant_tensor(v: sp.csr_matrix, lin_op_shape: tuple[int, int]) \
+            -> sp.csc_matrix:
+        p = np.prod(v.shape) // np.prod(lin_op_shape)
+        old_shape = (v.shape[0] // p, v.shape[1])
+
+        coo = v.tocoo()
+        data, stacked_rows, cols = coo.data, coo.row, coo.col
+        slice, rows = np.divmod(stacked_rows, old_shape[0])
```

Comment:
here slice is often just going to be a list of zeros. 
This is because old_shape[0] is the size of the vectorised data. Which will almost always be greater than the values in stacked_rows, since we are reshaping to a smaller shape.

Reply:
It will contain nonzero elements whenever we are dealing with a parameter.